### ISSUE TITLE
Can theme attributes be set for the new shapes in Power BI? 

### ISSUE THREAD

**Issue by steved-MarketGrunt** [MattRudy/PowerBI-ThemeTemplates#121]
I noticed the Json template file for shapes doesn't appear to work with the new shape styles. Is this your understanding as well?

**Comment by MattRudy** [comment]
Yes, there's an open issue to check them - there were two different names to reference shapes. I believe you need to change the term 'shape' to 'basicShape' as in the case below:

```
...
     "visualStyles": {
		"basicShape": {
		}
    }
...
````

**Comment by steved-MarketGrunt** [comment]
"basicShape" appears to work with unconverted shapes. "shape" which I assume is for the new format shapes doesn't appear to be doing much. It certainly doesn't change the fill and line colour from the default values.

BTW, I appreciate that you are doing this on your personal time so, let me know if I can be of assistance in resolving this issue. I'm happy to share any resolution back to the group.
